### ISSUE TITLE
Stuck on "Settings are still being loaded"

### ISSUE THREAD

**Issue by Akyariss** [FrogTheFrog/moondeck#27]
I am permanently stuck on "settings are still being loaded" and i have no settings folder or log file

**Comment by FrogTheFrog** [comment]
- Did you install this plugin via decky loader?
- Are you on stable?

Please give me more info about your environment since I can't help you otherwise...

**Comment by Akyariss** [comment]
Yes, i installed it via Decky loader and i am on the last stable version

**Comment by FrogTheFrog** [comment]
I will try to do a fresh install myself today, but in the meantime could you try to:

1. Uninstall MoonDeck
2. Restart deck
3. Install MoonDeck
4. Restart deck

It really seems like some permissions issue to me which should not happen since the MoonDeck does not need them...
